content/01.main_content.md

```diff
@@ -1,4 +1,15 @@
-## Abstract {.page_break_before}
 
-
-Add text here, example citation [@https://www.theguardian.com/world/2016/may/25/enduring-whiteness-of-american-journalism; @https://medium.com/ladybits-on-medium/i-analyzed-a-year-of-my-reporting-for-gender-bias-and-this-is-what-i-found-a16c31e1cdf#.44nchdhay]
\ No newline at end of file
+Computationally-derived predictions allow for diversity audits on a scale that would not be possible otherwise.
+  Numerous tools exist to probabilistically infer gender, nationality, and ethnicity information using only the feature most likely to be non-missing in a relevant dataset: an individual's name. <decide what to cite here based on how many sources we have left>
```

Comment:
maybe "algorithmically" instead of "probabilistically" to capture a larger set of methods of inference